lib/Driver/ToolChain.cpp

```diff
@@ -37,8 +38,8 @@ ToolChain::JobContext::JobContext(Compilation &C,
                                   ArrayRef<const Action *> InputActions,
                                   const CommandOutput &Output,
                                   const OutputInfo &OI)
-  : C(C), Inputs(Inputs), InputActions(InputActions), Output(Output),
-    OI(OI), Args(C.getArgs()) {}
+: C(C), Inputs(Inputs), InputActions(InputActions), Output(Output),
```

Comment:
This looks like it got mis-formatted. Or if it's what clang-format does, clang-format is clearly wrong!

Reply:
I must have manually set it back to correspond with a prior SHA: 26d6502.
(Hard to know when to leave git-clang-format alone vs when to avoid changes.)